client/extensions/woocommerce/app/order/order-details/index.js

```diff
@@ -62,7 +62,7 @@ class OrderDetails extends Component {
 		return [
 			<OrderCreated key="order-date" order={ order } site={ site } />,
 			<OrderDetailsTable key="order-details" order={ order } site={ site } />,
-			<OrderPaymentCard key="order-payment" order={ order } site={ site } />,
+			<OrderPaymentCard key="order-payment" order={ order } siteId={ site.ID || null } />,
```

Comment:
Why have OrderPaymentCard accept siteId instead of site like its siblings?

Reply:
We don't need the full site object, just ID. We talked about simplifying props at the GM, and not passing around entire objects when we don't need to -- so as I refactor some of these components, I'm also changing out the props.